### ISSUE TITLE
Actual language specs

### ISSUE THREAD

**Issue by christopherpeart** [covfefe-lang/covfefe#4]
We need that. Assigning to me and Dmirty

**Comment by christopherpeart** [comment]
I think making sure that names are at the optimal Covfefe level is important

**Comment by Destructible-Watermelon** [comment]
I think that how it should work is that you can only use words trump has used on twitter, and commands are the number of covfefiable words before a covfefonym: Despite the constant negative press covfefe would be command index 3

**Comment by anna328p** [comment]
I thought that I would covfefify actual language keywords. Both approaches are valid.

**Comment by christopherpeart** [comment]
I agree with ^^ but I think that you should have to always use covfefify words for variables and commands

**Comment by Destructible-Watermelon** [comment]
Also where should crooked hillary clinnono come into this?

**Comment by anna328p** [comment]
Maybe that could be the term for exceptions/errors

**Comment by Destructible-Watermelon** [comment]
ok, I am making an example hello world program I think. will report back later

**Comment by Destructible-Watermelon** [comment]
hmm, I started writing a program, but there is a fair amount lacking covfefe, but I still don't want there to be too much covfefe. however, I guess covfefe can be used especially with control flow commands

**Comment by Destructible-Watermelon** [comment]
Aha! I realised that I can simply allow variables to be no less than one, as well as comparisons being covfefe anyway. nvm lol. but anyway this hello world program might be a bit uncovfefey, but be assured that the finished product will be more covfefey

**Comment by Destructible-Watermelon** [comment]
https://gist.github.com/Destructible-Watermelon/84635a0434338ba78aadbceeeec9e8a9

no commas or capitals, but does include !. this lang doesn't have any actual key words I don't think. also I'm undecided whether the variable has to come at the start of a covfefe. also I'm not 100% sure all the words were on his twitter

**Comment by anna328p** [comment]
I was thinking more like `ifveve(varidada == 3){ print(pop); }`

**Comment by Destructible-Watermelon** [comment]
;_; I put so much effort into this!

I do think that it should resemble trump tweets rather than just having some covfefey commands. mine works in this way

I don't want this to be another arnoldc or lolcode that think funny function names / keywords = esoteric

**Comment by Destructible-Watermelon** [comment]
btw if you guys think that your covfefe keywords idea is better please tell me so I can work on my own covfefe lang alone, or as a separate repo here. possibly the covfefey keywords could be a new repo called covfefescript. IDK

P.S. Varidada is not covfefey

**Comment by anna328p** [comment]
Maybe we should make a stack based brainfuck?

**Comment by anna328p** [comment]
With covfefey variable names

**Comment by Destructible-Watermelon** [comment]
pls no

**Comment by anna328p** [comment]
pls yes

**Comment by anna328p** [comment]
OK, your idea is really good now that I'm not tired. The "number of covfefifiable words before a covfefonym" is very interesting. 

How would the commands be assigned? Would it just be by number, or something to do with the words?

**Comment by Destructible-Watermelon** [comment]
A bit of both; covfefe commands have the variable(s) they are acting on, but other than that it is just numbers. At least, it has worked like this for my program so far. Also there should probably be labels. How should labels be made? maybe during the same part as variable declaration, and referenced by number?

**Comment by anna328p** [comment]
"This is LAVEVE1"

**Comment by christopherpeart** [comment]
This is ded
